examples/vertx-opentracing/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,28 @@
+#
```

Comment:
Isn't it better to put it in example root folder? It does not belong to app source code.